pointmatcher/InspectorsImpl.cpp

```diff
@@ -320,25 +320,42 @@ void InspectorsImpl<T>::AbstractVTKInspector::dumpDataLinks(
 	}
 	const int knn = matches.ids.rows();
 	
-	stream << "LINES " << readingPtCount*knn << " "  << readingPtCount*knn * 3 << "\n";
+	size_t matchCount = readingPtCount*knn;
+	for (int k = 0; k < knn; ++k)
+	{
+		for (int i = 0; i < readingPtCount; ++i)
+		{
+			if (matches.ids(k, i) == Matches::InvalidId){
+				matchCount --;
+			}
+		}
+	}
+
+	stream << "LINES " << matchCount << " "  << matchCount * 3 << "\n";
 	//int j = 0;
```

Comment:
Dead code

Reply:
I didn't touch these old dead lines and I would leave general cleaning up to cleanup PRs (makes the reviewing easier)